coderabbit.yaml

```
@@ -0,0 +1,87 @@
+# yaml-language-server: $schema=https://storage.googleapis.com/coderabbit_public_assets/schema.v2.json
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)